docs/agile-methodologies.md

```diff
@@ -28,4 +29,137 @@ It consists of the elaboration of a table or diagram in which three columns of t
+3. Re-assessing the relative priority of stories.
+4. Assigning estimates to stories which have yet to receive one
+correcting estimates in light of newly discovered information.
+5. Splitting user stories which are high priority.
+6. Get back to work.
+
+### **Retrospective Session** 
+
+Is a meeting held after a product ships to discuss what happened during the product development and release process, with the goal of improving things in the future based on those learnings and conversations.  
+An agile retrospective forces the entire team to pause and reflect on what transpired and discuss what worked and what didn’t during a particular project.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/MiaZhJyYUj0" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How often should you hold retrospectives?
+
+Retrospectives can be held more frequently, including for minor releases, each sprint or even at daily or weekly standups.
+
+### **Demo Session**
+
+The sprint demo is invaluable for keeping stakeholders up to speed with the progress of product development. It allows them to feedback and discuss with the Product Owner and Scrum team any possible amendments to the Product Backlog which would help to maximize value.  
+The sprint demo shouldn’t take up too much of a Scrum team’s time. Ensuring that the sprint review meeting is an informal affair where questions, feedback and discussion are welcome – allows for prep time to be kept to a minimum.
+
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/Njxek3s3Shs" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Demo steps
+
+1. **Tell a story:** This is one of the most important factors for a great demo, and also the most overlooked. Given the structured nature of Agile stories and epics, it’s easy to fall into the trap of simply enumerating the work that you’ve done. This isn’t necessarily bad, but it’s unlikely to excite non-technical stakeholders.
+2. **Let developers brag:** Whenever possible, it’s great to allow developers to present their own work, which helps to build confidence, morale, and a sense of ownership. A good compromise can be to have one organizing speaker with a different “guest” developer showing off their work each week.
+3. **Set expectations:** Setting expectations and providing context are critical for a successful demo. 
+4. **Action items:** List all completed stories in your agenda, weed out any stories that shouldn’t be demoed, organize the remaining stories roughly into scenarios or topics, decide whether to have developers help give parts of the demo and always set expectations and give context throughout the demo.
+
+### **Stand-up Session**
+
+A daily stand-up meeting is an opportunity for the project team to discuss a project’s progress at a high level. These meetings last 15 minutes and allow each contributor to report on their accomplishments since the last stand-up meeting.
```

Comment:
Consider changing to : These meetings last approximately 15 minutes

Reply:
Thanks!!